dev/wlp-gradle/propertiesSettings.gradle

```
@@ -95,12 +95,13 @@ def setProperties = { props ->
      */
     String timestamp = new SimpleDateFormat("yyyyMMddHHmm").format(new java.util.Date())
 
-    // For release builds we use the buildLabel as a consistent qualifier.
+    // For automated builds we use the buildLabel as a consistent qualifier.
     // buildLabel is set by the RTC build engines.
-    String qualifier = (isRelease && isAutomatedBuild) ? props.getProperty('buildLabel') : timestamp
+    String buildLabel = props.getProperty('buildLabel')
+    String qualifier = isAutomatedBuild ? buildLabel : timestamp
```

```suggestion
    String qualifier = isAutomatedBuild ? props.getProperty('buildLabel') : timestamp
```
Since you aren't using the buildLabel in two places no need to store it any longer.